templates/index.php

```diff
@@ -75,7 +75,7 @@ class="
 						{{relativeModifiedDate dateInt}}
 					</span>
 			</div>
-			<div class="icon-delete action delete"></div>
+			<a class="icon-delete delete action" original-title="{{delete}}"></a>
```

Comment:
Don't use `original-title` here, just `title` (`original-title` is created by Tipsy, and if we switch to Bootstrap at some point it might change)